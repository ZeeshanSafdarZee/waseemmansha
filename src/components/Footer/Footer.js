import React from "react";
import { FiGithub, FiCodepen, FiLinkedin } from "react-icons/fi";
import { Heading, Flex, Box } from "../UI";
import FooterLink from "./FooterLink";

const Footer = React.memo(() => (
  <Box is="footer" py={58} bg="bgAlt" color="white">
    <Flex alignItems="center" justifyContent="center">
      {/* <FooterLink
        href="https://github.com/janczizikow/"
        target="_blank"
        rel="noopener noreferrer"
        aria-label="Github"
      >
        <FiGithub size={24} />
      </FooterLink>
      <FooterLink
        href="https://codepen.io/hollow3d/"
        target="_blank"
        rel="noopener noreferrer"
        aria-label="Codepen"
      >
        <FiCodepen size={24} />
      </FooterLink>
      <FooterLink
        href="https://www.linkedin.com/in/janczizikow/"
        target="_blank"
        rel="noopener noreferrer"
        aria-label="Linkedin"
      >
        <FiLinkedin size={24} />
      </FooterLink> */}
      <Heading color="white">
        Waseem Mansha Full Stack Mobile/Web Developer
      </Heading>
      <Heading color="white">
        Global Software Consulting 457 Johar Town Lahore Lahore,
      </Heading>
    </Flex>
  </Box>
));

export default Footer;
