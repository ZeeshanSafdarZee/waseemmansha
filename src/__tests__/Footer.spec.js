import toJson from 'enzyme-to-json';
import Footer from '../components/Footer';

describe('<Footer />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Footer />);
  });

  it('renders without crashing', () => {
    expect(wrapper).toHaveLength(1);
  });

  it('renders with correct styles', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
