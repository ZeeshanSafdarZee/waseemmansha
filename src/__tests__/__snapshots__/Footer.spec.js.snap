// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders with correct styles 1`] = `
<Box
  bg="bgAlt"
  blacklist={
    Array [
      "m",
      "mt",
      "mr",
      "mb",
      "ml",
      "mx",
      "my",
      "p",
      "pt",
      "pr",
      "pb",
      "pl",
      "px",
      "py",
      "width",
      "fontSize",
      "color",
      "bg",
      "fontFamily",
      "textAlign",
      "lineHeight",
      "fontWeight",
      "fontStyle",
      "letterSpacing",
      "display",
      "maxWidth",
      "minWidth",
      "height",
      "maxHeight",
      "minHeight",
      "size",
      "ratio",
      "verticalAlign",
      "alignItems",
      "alignContent",
      "justifyItems",
      "justifyContent",
      "flexWrap",
      "flexBasis",
      "flexDirection",
      "flex",
      "justifySelf",
      "alignSelf",
      "order",
      "gridGap",
      "gridColumnGap",
      "gridRowGap",
      "gridColumn",
      "gridRow",
      "gridAutoFlow",
      "gridAutoColumns",
      "gridAutoRows",
      "gridTemplateColumns",
      "gridTemplateRows",
      "gridTemplateAreas",
      "gridArea",
      "border",
      "borderTop",
      "borderRight",
      "borderBottom",
      "borderLeft",
      "borderColor",
      "borderRadius",
      "boxShadow",
      "opacity",
      "overflow",
      "background",
      "backgroundImage",
      "backgroundPosition",
      "backgroundRepeat",
      "backgroundSize",
      "position",
      "zIndex",
      "top",
      "right",
      "bottom",
      "left",
      "textStyle",
      "colors",
      "variant",
      "theme",
    ]
  }
  color="white"
  is="footer"
  py={58}
>
  <Flex
    alignItems="center"
    display="flex"
    justifyContent="center"
  >
    <FooterLink
      aria-label="Github"
      href="https://github.com/janczizikow/"
      rel="noopener noreferrer"
      target="_blank"
    >
      <FiGithub
        size={24}
      />
    </FooterLink>
    <FooterLink
      aria-label="Codepen"
      href="https://codepen.io/hollow3d/"
      rel="noopener noreferrer"
      target="_blank"
    >
      <FiCodepen
        size={24}
      />
    </FooterLink>
    <FooterLink
      aria-label="Linkedin"
      href="https://www.linkedin.com/in/janczizikow/"
      rel="noopener noreferrer"
      target="_blank"
    >
      <FiLinkedin
        size={24}
      />
    </FooterLink>
  </Flex>
</Box>
`;
